heron/spi/src/java/com/twitter/heron/spi/common/ClusterConfig.java

```diff
@@ -25,137 +25,60 @@
   private ClusterConfig() {
   }
 
-  private static Config loadHeronHome(String heronHome, String configPath) {
-    Config.Builder cb = Config.newBuilder()
+  private static Config loadDefaults(String heronHome, String configPath) {
+    return Config.newBuilder(true)
```

Comment:
True says to initialize the Config with default values. See `Config.Builder.create(loadDefaults)` which this call delegates to.